instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/config/Config.java

```diff
@@ -159,4 +159,8 @@ public Properties asJavaProperties() {
     properties.putAll(getAllProperties());
     return properties;
   }
+
+  public boolean isAgentDebugEnabled() {
+    return getBooleanProperty("otel.javaagent.debug", false);
```

Comment:
Can we get a test to cover this?  Might seem silly, but it helps to confirm the key name and also the default value.

Reply:
done